src/MovieLogger.jsx

```diff
@@ -31,9 +30,6 @@ function MovieLogger() {
       body: JSON.stringify({ userId, movieId, rating }),
     })
       .then((response) => response.json())
-      .then((data) => {
```

Comment:
While we don't need to console.log that the Movie successfully got logged. It would bo good practice to notify the user of the success.